ShoppingCommonProtos/src/Type/CustomAttribute.php

```diff
@@ -0,0 +1,164 @@
+<?php
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: google/shopping/type/types.proto
```

Comment:
Should this have Google License?

Reply:
Same for other 2 files in `src`

Reply:
These are autogenerated and the same for all protobuf message files in this library